frontend/src/scenes/feedback/feedbackLogic.ts

```diff
@@ -0,0 +1,69 @@
+import { actions, afterMount, kea, path, reducers, selectors } from 'kea'
+import { loaders } from 'kea-loaders'
+import api from 'lib/api'
+import { EventType } from '~/types'
+
+import type { feedbackLogicType } from './feedbackLogicType'
+
+export const feedbackLogic = kea<feedbackLogicType>([
+    path(['scenes', 'feedback', 'feedbackLogic']),
+    actions({
+        setTab: (activeTab: string) => ({ activeTab }),
+        toggleInAppFeedbackInstructions: true,
+        setExpandedSection: (idx: number, expanded: boolean) => ({ idx, expanded }),
+    }),
+    reducers({
+        activeTab: [
+            'in-app-feedback' as string,
+            {
+                setTab: (_, { activeTab }) => activeTab,
+            },
+        ],
+        inAppFeedbackInstructions: [
+            false,
+            {
+                toggleInAppFeedbackInstructions: (state) => !state,
+            },
+        ],
+        expandedSections: [
+            [true, false] as boolean[],
+            {
+                setExpandedSection: (state, { idx, expanded }) => {
+                    // set all to false apart from the one we're changing
+                    return state.map((_, i) => (i === idx ? expanded : false))
+                },
+            },
+        ],
+        events: [
```

Comment:
I don't think this is needed? IIRC the loaders plugin makes sure the reducer and actions are all created i.e. your `events` definition below is enough

Reply:
good spot